src/sonic-config-engine/minigraph.py

```diff
@@ -1458,6 +1458,28 @@ def select_mmu_profiles(profile, platform, hwsku):
                 base_file = os.path.join(path, file_item)
                 exec_cmd(["sudo", "cp", file_in_dir, base_file])
 
+def update_forced_mgmt_route(mgmt_intf, mgmt_routes):
+    for mgmt_intf_key in mgmt_intf.keys():
+        forced_mgmt_routes = []
+
+        # get mgmt interface type
+        mgmt_intf_addr = mgmt_intf_key[1]
+        mgmt_is_ipv4 = True
+        if ":" in mgmt_intf_addr:
```

Comment:
>if ":" in mgmt_intf_addr: [](http://example.com/codeflow?start=8&length=25)

Suggest using library like ipaddress.ip_address to check ipv4/ipv6.

Reply:
Fixed, use ipaddress.ip_network because this can handle both address and address block

Reply:
ipaddress class not exist in python2, if we want backport, still need check ':' exist.